### DISCUSSION TITLE
Connecting to Huawei inverter

### DISCUSSION THREAD

**Discussion by IngoAlta** [wlcrs/huawei_solar#454] [Setting up the integration]
I am new to this. I fist started Home Assistant on a RPI 4 and now I have changed to a Intel NUC 10i5 with 64 GB RAM and 1 TB SSD and installed Home Assistant. I followed the Generic x86-64 and after that install my backups from the old RPI installation. It looks like all is working fine. 
Now I am trying to install the Wuawei Solar integration with my SUN2000-8KTL-M1 inverter. I have installed the integration through the IP provided by the SDongelA-05. That don’t work so well. The connection is interrupted very frequent.
 So, I mover my NUC closer to the inverter and tried to connect to the inverters access point through the NUC’s built in WiFi. That is where I get lost. I followed the guide “Connecting to the inverter”  - “Getting connectivity between HA on your home network and the inverter AP”. 
I believe that the NUC is connected to the inverters AP but I really do not know how to check that. The guide say “You can configure this device in Hass.io via Configuration -> Add-ons, Backups & Supervisor -> System -> Change (next to IP address) -> WLAN<x>.” >.”  I can’t find that on the UI and do not know how to get there. Maybe through the Terminal or through UI Settings – System Network where I think I connected to the inverter.  Could this be right? 
![Config](https://github.com/wlcrs/huawei_solar/assets/113201564/01080aa1-c700-4969-a6c9-6ad2b4ef170d)

Next problem is the Ping command which I can’t get to. When I use the Terminal I get this message: 
![Ping](https://github.com/wlcrs/huawei_solar/assets/113201564/5f80a2bb-ea38-4374-b496-e6fe0ba91dd1)
I might be missing some important steps without knowing that. Can someone guide me right with these issues? Thankfull for all help I can get.

**Comment by Roving-Ronin** [comment]
Connect your inverters WiFi/FE dongle to your home network's WiFi AP, assign it a static IP (reserved IP in DHCP) and then connect to the inverter over your home network.
